src/main/kotlin/pt/up/fe/ni/website/backend/config/security/AuthConfig.kt

```diff
@@ -60,7 +60,7 @@ class AuthConfig(
 
     fun rolesConverter(): JwtAuthenticationConverter? {
         val authoritiesConverter = JwtGrantedAuthoritiesConverter()
-        authoritiesConverter.setAuthorityPrefix("ROLE_")
+        authoritiesConverter.setAuthorityPrefix("")
```

Comment:
why keep this line?

Reply:
It's easier to serialize the roles if it has a prefix

Reply:
What's the difference between no prefix and an empty string?

Reply:
If this line is not in the code, Spring will give a default prefix to the role, with this line, there is not going to be a prefix.